src/runtime_src/core/tools/xbmgmt2/OO_Config.cpp

```diff
@@ -131,8 +131,10 @@ show_device_conf(xrt_core::device* device)
 
   try {
     auto is_mfg = xrt_core::device_query<xrt_core::query::is_mfg>(device);
-    if (is_mfg) {
-      throw std::runtime_error("This operation is not supported with golden image");
+    auto is_recovery = xrt_core::device_query<xrt_core::query::is_recovery>(device);
```

Comment:
All devices will have this sysfs node correct?

To not break windows, can you add the following in core/pcie/windows/device_windows.cpp
```
struct recovery
{
  using result_type = bool;

  static result_type
  user(const xrt_core::device* device, key_type key)
  {
    return false;
  }

  static result_type
  mgmt(const xrt_core::device* device, key_type key)
  {
    return false;
  }
};
```

In L1094:
`emplace_function0_getter<query::is_recovery,           recovery>();`

This hard codes recovery ioctl to false until we can get the actual value from the windows driver

Reply:
> struct recovery
> {
>   using result_type = bool;
> 
>   static result_type
>   user(const xrt_core::device* device, key_type key)
>   {
>     return false;
>   }
> 
>   static result_type
>   mgmt(const xrt_core::device* device, key_type key)
>   {
>     return false;
>   }
> };

I will blandly add those code. Because I don't have a windows build environment.

Reply:
@AShivangi  windows build failed.